src/screens/Assistant/journey-date-picker/index.tsx

```diff
@@ -0,0 +1,154 @@
+import Button from '@atb/components/button';
+import ScreenHeader from '@atb/components/screen-header';
+import {
+  DateInputItem,
+  RadioSection,
+  Section,
+  TimeInputItem,
+} from '@atb/components/sections';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {JourneyDatePickerTexts, useTranslation} from '@atb/translations';
+import {dateWithReplacedTime, formatLocaleTime} from '@atb/utils/date';
+import {
+  NavigationProp,
+  ParamListBase,
+  RouteProp,
+  useRoute,
+} from '@react-navigation/core';
+import React, {useRef, useState} from 'react';
+import {ScrollView} from 'react-native';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import {AssistantParams} from '..';
+
+export type DateTimePickerParams = {
+  searchTime: SearchTime;
+  callerRouteName: string;
+  callerRouteParam: string;
+};
+
+export type DateTimeNavigationProp = NavigationProp<AssistantParams>;
+export type DateTimeRouteProp = RouteProp<AssistantParams, 'DateTimePicker'>;
```

Comment:
I get this in the console, even though I can not see by just reviewing the code what is causing it. 

>[Wed Feb 24 2021 11:10:57.479]  WARN     Non-serializable values were found in the navigation state. Check:
>
>TabNavigator > Assistant > DateTimePicker > params.onChange (Function)
>
>This can break usage such as persisting and restoring state. This might happen if you passed non-serializable values such as function, class instances etc. in params. If you need to use components with callbacks in your options, you can use 'navigation.setOptions' instead. See https://reactnavigation.org/docs/troubleshooting#i-get-the-warning-non-serializable-values-were-found-in-the-navigation-state for more details.

Reply:
I think you are running old code here. This should have been fixed by c6a3c5a